eth2near/eth2near-block-relay-rs/src/beacon_rpc_client.rs

```diff
@@ -51,11 +48,11 @@ impl BeaconRPCClient {
             client: reqwest::blocking::Client::builder()
                 .timeout(Duration::from_secs(timeout_seconds))
                 .build()
-                .unwrap(),
+                .expect("Error on building blocking client for regular rpc requests."),
             client_state_request: reqwest::blocking::Client::builder()
                 .timeout(Duration::from_secs(timeout_state_seconds))
                 .build()
-                .unwrap(),
+                .expect("Error on building blocking client for state request."),
         }
```

Comment:
This panic as well.

Reply:
Here it is Okey, because it is during initialization(one time at the start). So, in that case, we want the program to finish.